crates/sc-consensus-subspace/src/archiver.rs

```diff
@@ -826,7 +840,12 @@ where
     Ok(async move {
         let archiver = match maybe_archiver {
             Some(archiver) => archiver,
-            None => initialize_archiver(&segment_headers_store, &subspace_link, client.as_ref())?,
+            None => initialize_archiver(
+                &segment_headers_store,
+                &subspace_link,
+                client.as_ref(),
+                force_mapping,
```

Comment:
I wonder if this should always be `false`, so that we pick up the mappings in any partial block that hasn't already been archived?

(If blocks end up being skipped, then we'll re-initialise the archiver without creating or sending those mappings.)

```suggestion
                false,
```

Reply:
I don't think it should, using a variable consistently is fine